### ISSUE TITLE
Use GitHub issues?

### ISSUE THREAD

**Issue by sdougherty** [KarenLindokken/LIMRA-EDI-Enrollment-Standard#1]
Should we use GitHub issues for raising questions, tracking issues, suggesting features and soliciting feedback? Issues would not replace the wiki.

**Comment by LyleGriffin** [comment]
Seems like the right approach to me. This would allow all active technical contributors to participate in the discussion. We can always share a summary of the Issues discussions with the committee at large during our regular meetings.

**Comment by KarenLindokken** [comment]
Agreed.  Having a central place we can view/edit/comment will facilitate communications and as we are spread across many time zones, allow folks to work on their own schedules yet still be apart of the group

**Comment by timoconnorvoya** [comment]
I've started getting our content into GitHub, specifically...

- Created a Project to track open items as Kanban cards.  Not the most elegant PM solution, but it should get the job done for the next two weeks
- Agree that any open discussion items should be logged as Issues so we can discuss
- Updated the Wiki with our meeting notes and discussions.  May also see if we can get post project artifacts such as the RFP, Data Model, UML Diagram, etc. out here
